### ISSUE TITLE
[Feat] Drop shadow option for shapes

### ISSUE THREAD

**Issue by KyleKlus** [excalidraw/excalidraw#7661]
It would be nice if you would give the user an option for adding a drop shadow to shapes.

I know, that this would deviate a bit from the excalidraw style, that everything looks hand drawn, 
but it would really come in handy when one is quickly sketching up some ui elements for a user interface,
where usually a lot of shadows are used for highlighting visual hierarchy.

**Comment by Archonic944** [comment]
This would be great for images. I would actually argue that it would help keep from the style better, as it would indicate that these non-hand drawn elements (images) are on a plane other the hand-drawn elements.

**Comment by CamilleHbp** [comment]
I agree with this request. Especially since the official `Use-cases showcase` has custom-made shadows for all components. It would be wonderful to drop a shadow of any colour/opacity for shapes.
